roles/manage_cnp/README.md

```diff
@@ -0,0 +1,98 @@
+# manage_cnp
+
+This Ansible Galaxy Role contains task files that assist in the management of CloudNativePG and EDB Postgres for Kubernetes.
+
+**The ansible playbook must be executed under an account that has full
+privileges.**
+
+## Requirements
+
+The only dependencies required for this ansible galaxy role are:
+
+  1. AWS EKS CLI, Azure CLI, or Google Cloud CLI - Depends directly on the target cloud
+  2. Kubectl
+  3. Ansible
+  4. Python packages: openshift, pyyaml, kubernetes, pyhelm
+     Installed with commands:
+     `pip install openshift pyyaml kubernetes`
+     `pip install pyhelm`
+  5. `kubernetes.core.k8s` - Ansible Module - Required for Kubernetes.
+
+## Role variables
+
+The 'cnp_task' must be defined with a value in order to indicate which task in the role will be executed.
+
+
+## Dependencies
+
+The `manage_cnp` role does not have any dependencies on any other roles.
+
+## Example Playbook
+
+## How to include the `manage_cnp` role in your Playbook
+
+Below is an example of how to include the `manage_cnp` role:
+
+```yaml
+---
+- name: Manage a CloudNativePG or CloudNative Postgres Cluster Playbook
+  hosts: localhost
+  gather_facts: false
+
+  vars:
+    # Options"
+    # add-db, add-schema, add-role
+    # drop-db, drop-schema, drop-role
+    - cnp_task: "add-db"
```

Comment:
same as comment on the playbook-examples

Reply:
converted to a list